Example/Messaging/Tests/FIRMessagingServiceTest.m

```diff
@@ -211,7 +228,8 @@ - (void)testUnsubscribeWithInvalidTopic {
 }
 
 - (void)testSubscribeWithNoTopicPrefix {
-
+  OCMStub([_mockInstanceID
```

Comment:
If this stub is needed for all tests, it may make sense to move its configuration to `setUp`.

Reply:
Yeah, OCMock may behave weirdly when stubbing the same method several times. LGTM as is.